addons/l10n_fi_rapko/data/account_chart_template_pre_data.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="account.account.template" id="a19190">
+        <field name="code">19190</field>
+        <field name="user_type_id" ref="account.data_account_type_receivable"/>
+        <field name="reconcile" eval="True"/>
+        <field name="name">Rahansiirrot/ v&#228;litilit/ t&#228;sm&#228;ytykset</field>
+    </record>
+
+    <!-- Account Chart Template -->
+    <record id="chart_template_fi_rapko" model="account.chart.template">
```

Comment:
Only the strings visible to the vendors, customers and other external parties are translated, ie. taxes. I don't think a proper translation exists for this CoA.